src/paperless_xml/signals.py

```diff
@@ -0,0 +1,16 @@
+def get_parser(*args, **kwargs):
+    from paperless_xml.parsers import XMLDocumentParser
+
+    return XMLDocumentParser(*args, **kwargs)
+
+
+def xml_consumer_declaration(sender, **kwargs):
+    return {
+        "parser": get_parser,
+        "weight": 11,
+        "mime_types": {
+            "text/plain": ".txt",
```

Comment:
Why do you want to handle text files with this extension?

Reply:
Because sometimes (at least in my Test) Firefox used text/plain when uploading XML. If it was not detected as an xinvoice XML the uaual text-parser will be used anyways

Reply:
Noticed that too with my project - as far as I can tell this seems to always happen when the file is missing the xml header (`<?xml version="1.0" encoding="UTF-8"?>`). But if that's the reason here too, I honestly would consider that an error on the invoices side, which is why I did not add text/plain to the signals file in my project.